ios/assets/assets/langs/en.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
Can you remove `en.json` and other localizations from ios assets`?

Reply:
comparing to this `master` branch,` ios/assets` folder is not exist. so removing `ios/assets` folder including en.json ?

Reply:
Yes I think we need to add this ti girignore. This will change once we generate new ios js bundle by running yarn  build:ios

Reply:
resolved by [f9e2b0e](https://github.com/dooboolab/hackatalk-mobile/pull/115/commits/f9e2b0e4a6ede2e9fe761d32f81e4e6ca5056bd2)